umm-spec-lib/test/cmr/umm_spec/test/validation/platform.clj

```diff
@@ -31,7 +40,17 @@
           (h/assert-invalid
             coll
             [:Platforms]
-            ["Platforms must be unique. This contains duplicates named [P1]."])))
+            ["Platforms must be unique. The combination of platform ShortName with Characteristics Name and Value contain duplicates named [P1]."])))
+      (testing "duplicate platform short names with characteristics"
+        (let [coll (coll/map->UMM-C
+                     {:Platforms [(c/map->PlatformType {:ShortName "DHC-6"
+                                                        :Characteristics [c3]})
+                                  (c/map->PlatformType {:ShortName "DHC-6"
+                                                        :Characteristics [c3]})]})]
+          (h/assert-invalid
+            coll
+            [:Platforms]
+            ["Platforms must be unique. The combination of platform ShortName with Characteristics Name and Value contain duplicates named [DHC-6AircraftIDC-FSJB]."])))
```

Comment:
I'm fine with merging this in since there is urgency for NSIDC. We should write a separate clean up ticket to display the ShortName, Characteristics Name and Value as provided in the metadata rather than a single concatenated string of all 3 (DHC-6AircraftIDC-FSJB).